tests/components/modbus/test_modbus_read_cache.py

```diff
@@ -0,0 +1,91 @@
+"""Test modbus read cache."""
```

Comment:
This should be part of the bit_sensor / bit_switch test, we do not want to caching generally.